src/gateway/router/routes/deployBundledRoute.ts

```diff
@@ -69,14 +68,7 @@ export async function deployBundledRoute(ctx: Router.RouterContext) {
     };
 
     await gatewayDb('contracts').insert(insert);
-
-    sleep(2000)
-      .then(() => {
-        sendNotificationToCache(ctx, bundlrResponse.data.id, initState);
-      })
-      .catch((e) => {
-        logger.error(`No sleep 'till Brooklyn.`, e);
-      });
+    sendNotification(ctx, bundlrResponse.data.id, {initState, tags: dataItem.tags});
```

Comment:
I believe that dataItem's tags are also already decoded - they are used in the `verifyContractTags` without additional decoding